content/es/docs/tasks/configure-pod-container/configure-volume-storage.md

```diff
@@ -0,0 +1,131 @@
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Configura un Volume para un Pod
+
+En este ejercicio crearás un Pod que ejecuta un único Contenedor. Este Pod tiene un Volume de tipo [emptyDir](/docs/concepts/storage/volumes/#emptydir) (directorio vacío) que existe durante todo el ciclo de vida del Pod, incluso cuando el Contenedor es destruído y reiniciado. Aquí está el fichero de configuración del Pod:
+
+{{< codenew file="pods/storage/redis.yaml" >}}
+
+1. Crea el Pod:
+
+    ```shell
+    kubectl apply -f https://k8s.io/examples/pods/storage/redis.yaml
+    ```
+
+1. Verifica que el Contenedor del Pod se está ejecutando y después observa los cambios en el Pod
+
+    ```shell
+    kubectl get pod redis --watch
+    ```
+
+    La salida debería ser similar a:
+
+    ```shell
+    NAME      READY     STATUS    RESTARTS   AGE
+    redis     1/1       Running   0          13s
+    ```
+
+1. En otra terminal, abre una shell dentro del Contenedor que se está ejecutando:
+
+    ```shell
+    kubectl exec -it redis -- /bin/bash
+    ```
+
+1. En la shell, ve a `/data/redis` y crea un fichero:
+
+    ```shell
+    root@redis:/data# cd /data/redis/
+    root@redis:/data/redis# echo Hello > test-file
+    ```
+
+1. En la shell, lista los procesos en ejecución:
+
+    ```shell
+    root@redis:/data/redis# apt-get update
+    root@redis:/data/redis# apt-get install procps
+    root@redis:/data/redis# ps aux
+    ```
+
+    La salida debería ser similar a:
+
+    ```shell
+    USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
+    redis        1  0.1  0.1  33308  3828 ?        Ssl  00:46   0:00 redis-server *:6379
+    root        12  0.0  0.0  20228  3020 ?        Ss   00:47   0:00 /bin/bash
+    root        15  0.0  0.0  17500  2072 ?        R+   00:48   0:00 ps aux
+    ```
+
+1. En la shell, mata el proceso Redis:
+
+    ```shell
+    root@redis:/data/redis# kill <pid>
+    ```
+
+    donde `<pid>` es el ID de proceso (PID) de Redis.
+
+1. En la terminal original, observa los cambios en el Pod de Redis. Eventualmente verás algo como lo siguiente:
+
+    ```shell
+    NAME      READY     STATUS     RESTARTS   AGE
+    redis     1/1       Running    0          13s
+    redis     0/1       Completed  0         6m
+    redis     1/1       Running    1         6m
+    ```
+
+En este punto, el Contenedor ha sido destruido y reiniciado. Esto es debido a que el Pod de Redis tiene una
+[restartPolicy](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core) (política de reinicio)
+de `Always` (siempre).
+
+1. Abre una shell en el Contenedor reiniciado:
```

Comment:
Quiza sea regionalismo pero siempre he conocido `shell` como masculino y no como femenino.

Reply:
Revisándolo me surgen varias dudas al respecto:

* Quizá se debería traducir el término "shell" por terminal en lugar de tomar la palabra prestada del inglés.
* En todo el texto he usado la forma femenina tanto de _terminal_ como de _shell_ cuando parece que la predominante en este contexto, al menos para terminal, es la forma masculina (http://lema.rae.es/dpd/srv/search?key=terminal):

`b) Se usa asimismo en ambos géneros, con predominio del masculino, cuando significa ‘dispositivo conectado a un ordenador o computadora central, al que puede enviar y del que puede recibir información’: «La biblioteca contará con su banco de datos y doscientos terminales de ordenador» (Abc [Esp.] 23.8.89); «La Justicia Electoral asignará a cada apoderado que lo solicite una terminal de computadora» (Abc [Par.] 14.11.96).`

¿Qué opináis @electrocucaracha @raelga?

Reply:
Me parece mejor la idea de traducir `shell` por `terminal` o `consola`, ya que *shell* podría hacer referencia al lenguaje de scripting. De tal manera que la traducción quedaría:

```
1. Abre una terminal en el Contenedor reiniciado:
```

Reply:
Cambiado todo el documento a "terminal", en género masculino. Me he decantado por la forma en masculino porque según la definición es la forma predominante. Personalmente, cualquiera de las dos me suena correcta.

Commit e663e87be569270297e521052e14229524d0d3a4

Reply:
@lominorama me parece que es mas usado en su forma femenina, por ejemplo la terminal de windows.

https://elportaldetuvoz.wordpress.com/tag/el-terminal-o-la-terminal/

Reply:
Yo coincido con @lominorama, en el caso de terminal dentro del marco de la computación, la RAE recomienda el masculino. El enlace que comenta @electrocucaracha, hace referencia al terminal dentro del ámbito del transporte, y tanto el blog como la RAE, coinciden con que en ese caso, el género predominante es el femenino.

_terminal @ http://lema.rae.es/dpd/srv/search?key=terminal_

> **terminal**. Como sustantivo, aunque vacila en género en todas sus acepciones, debe tenerse en cuenta lo siguiente:
>
> b) Se usa asimismo en ambos géneros, **con predominio del masculino, cuando significa ‘dispositivo conectado a un ordenador o computadora central, al que puede enviar y del que puede recibir información’**: «La biblioteca contará con su banco de datos y doscientos terminales de ordenador» (Abc [Esp.] 23.8.89); «La Justicia Electoral asignará a cada apoderado que lo solicite una terminal de computadora» (Abc [Par.] 14.11.96).
>
> c) Cuando significa ‘conjunto de instalaciones situadas en el extremo de una línea de transporte o comunicación, o de un oleoducto’, se usa predominantemente en femenino en la mayor parte del ámbito hispánico: «Al bajarme del autobús en la terminal de Ticomán sentí el calorón» (Ibargüengoitia Crímenes [Méx. 1979]); pero en algunos países como Chile, Colombia, el Perú o Venezuela se prefiere claramente el masculino: «Al ingresar en el terminal lo vi charlando animadamente con el jefe de estación» (Collyer Pájaros [Chile 1995]).

Reply:
Okay, quien soy yo para cuestionar a la RAE